packages/template-retail-react-app/app/pages/account/order-detail.jsx

```diff
@@ -162,9 +229,9 @@ const AccountOrderDetail = () => {
                                     </Text>
                                     <Box>
                                         <Text fontSize="sm" textTransform="titlecase">
-                                            {shippingStatus.replace(/_/g, ' ')}
+                                            {shippingStatus?.replace(/_/g, ' ')}
```

Comment:
What is the content of `shippingStatus`. Is this something that needs to be extracted to an intl message?

Reply:
Although I think it's outside of the scope of the integration hooks work (the logic is what we use today in `develop`), I think is a good catch, and easy enough to fix it here. 

We're using now `formatMessage` to render that string.

These are the possible values:
```
not_shipped
part_shipped
shipped
```

SFRA Docs: https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-orders?meta=getOrder